server/routes/sendEmail.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const sgMail = require("@sendgrid/mail");
+require("dotenv").config();
+sgMail.setApiKey(process.env.sendGridKey);
+
+//dummy message for now. we can think about templates etc once we have models, controller set up 
+const msg = {
+  to: "vicscherman@gmail.com",
+  from: "hatchwayssparta@gmail.com",
+  subject: "With love from TattooArt!",
+  text: "This is a test email sent from our back end server",
+};
+
+router.post("/sendemail", (req, res) => {
```

Comment:
You are not returning a response from this route, that's why in your provided video the request continues to be pending even after the email is sent